### PR TITLE
release

### PR DESCRIPTION
**Pull request by Slavacheban**

### CODE REVIEW

File: src/main/java/HW14/abstractFactory/individs/canFly/MarsFlyer.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package HW14.abstractFactory.individs.canFly;
+
+public class MarsFlyer implements FlyAble {
+    @Override
+    public void fly() {
+    }
```
Comment (Taniann) [inline src/main/java/HW14/abstractFactory/individs/canFly/MarsFlyer.java]: why this method is empty?

File: src/main/java/HW14/abstractFactory/individs/canFly/PeopleFlyer.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package HW14.abstractFactory.individs.canFly;
+
+public class PeopleFlyer implements FlyAble {
+    @Override
+    public void fly() {
+    }
```
Comment (Taniann) [inline src/main/java/HW14/abstractFactory/individs/canFly/PeopleFlyer.java]: better write something in methods to see then how abstract factory works

File: src/main/java/HW14/factoryMeth/patterns/Builder.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package HW14.factoryMeth.patterns;
+
+public class Builder implements Pattern{
```
Comment (Taniann) [inline src/main/java/HW14/factoryMeth/patterns/Builder.java]: why is it empty?

File: src/main/java/HW14/abstractFactory/factories/AbstractFactory.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package HW14.abstractFactory.factories;
```
Comment (Taniann) [inline src/main/java/HW14/abstractFactory/factories/AbstractFactory.java]: package names in lowercase

File: src/main/java/HW14/protitype/Copiable.java [hunk 5]
```diff
@@ -0,0 +1,5 @@
+package HW14.protitype;
```
Comment (Taniann) [inline src/main/java/HW14/protitype/Copiable.java]: prototype

File: src/main/java/HW14/factoryMeth/patterns/Proxy.java [hunk 6]
```diff
@@ -0,0 +1,4 @@
+package HW14.factoryMeth.patterns;
+
+public class Proxy implements Pattern {
```
Comment (Taniann) [inline src/main/java/HW14/factoryMeth/patterns/Proxy.java]: why is it empty

File: src/main/java/HW14/factoryMeth/patterns/Singleton.java [hunk 7]
```diff
@@ -0,0 +1,4 @@
+package HW14.factoryMeth.patterns;
+
+public class Singleton implements Pattern {
```
Comment (Taniann) [inline src/main/java/HW14/factoryMeth/patterns/Singleton.java]: the same

**Review by Taniann — CHANGES_REQUESTED** [review]
please resolve comments
and I didnt see adstract factory producer
builder pattern

File: src/main/java/HW14/abstractFactory/individs/canFly/PeopleFlyer.java (hunk 2 above)
Comment (Slavacheban) [reply src/main/java/HW14/abstractFactory/individs/canFly/PeopleFlyer.java]: we dont use "sout"...  what can i write without sout?

File: src/main/java/HW14/protitype/Copiable.java (hunk 5 above)
Comment (Slavacheban) [reply src/main/java/HW14/protitype/Copiable.java]: ))

File: src/main/java/HW14/abstractFactory/individs/canFly/MarsFlyer.java (hunk 1 above)
Comment (Slavacheban) [reply src/main/java/HW14/abstractFactory/individs/canFly/MarsFlyer.java]: what means "resolve comments?" haw can i resolve comments?

File: src/main/java/HW14/abstractFactory/individs/canFly/PeopleFlyer.java (hunk 2 above)
Comment (Taniann) [reply src/main/java/HW14/abstractFactory/individs/canFly/PeopleFlyer.java]: > we dont use "sout"... what can i write without sout?

why dont we use sout?

File: src/main/java/HW14/abstractFactory/individs/canFly/MarsFlyer.java (hunk 1 above)
Comment (Taniann) [reply src/main/java/HW14/abstractFactory/individs/canFly/MarsFlyer.java]: > what means "resolve comments?" haw can i resolve comments?

make fixing according to comments

Comment (Slavacheban) [reply src/main/java/HW14/abstractFactory/individs/canFly/MarsFlyer.java]: закоммитить коментарии?

File: src/main/java/HW14/abstractFactory/individs/canFly/PeopleFlyer.java (hunk 2 above)
Comment (Slavacheban) [reply src/main/java/HW14/abstractFactory/individs/canFly/PeopleFlyer.java]: ну так решили) сейчас сделаю, а по поводу builder singleton там ж не сказанно реализовать паттерны. и я не представляю как билдер вернуть в виде типа

Comment (Slavacheban) [reply src/main/java/HW14/abstractFactory/individs/canFly/PeopleFlyer.java]: короче сделал, пока ты отмываешь новую работу)

**Review by Taniann — APPROVED** [review]
good

**Pull request merged**